sdks/python/test-suites/portable/py2/build.gradle

```diff
@@ -29,7 +29,7 @@ addPortableWordCountTasks()
 
 task preCommitPy2() {
   dependsOn ':runners:flink:1.5:job-server-container:docker'
-  dependsOn ':sdks:python:container:docker'
+  dependsOn ':sdks:python:container:buildDocker'
```

Comment:
Could we only depend on ':sdks:python:container:py2:docker rather than build all images? Same below.